strangerstudios/paid-memberships-pro
# Edit Level's Categories: Show a Message for Advanced Setting

### All Submissions:


### Changes proposed in this Pull Request:
In an effort to help site owners understand how their restricted categories will display on the front end, this update add a notice explaining how the Advanced Settings for filtering queries and showing excerpts are set.

### How to test the changes in this Pull Request:

1. Edit a Membership Level
2. Look for message below the "Content Settings" section.
3. Change your Memberships > Settings > Advanced Settings to NOT filter queries. Check the updated message.
4. Change your Memberships > Settings > Advanced Settings to hide excerpts. Check the updated message.
5. Change your Memberships > Settings > Advanced Settings to show excerpts. Check the updated message.

### Other information:


### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

* ENHANCEMENT: Added message to help site admin understand how queries and post content will be affected when restricting a category by membership level on the Edit Level page.
